include/RED4ext/GameEngine.hpp

```diff
@@ -218,19 +221,19 @@ struct CGameEngine : BaseGameEngine
 
     static CGameEngine* Get();
 
-    int64_t unk2F0;            // 2F0
-    int64_t unk2F8;            // 2F8
-    int32_t unk300;            // 300
-    int32_t unk304;            // 304
-    CGameFramework* framework; // 308
-    int64_t unk310;            // 310
-    int64_t unk318;            // 318
-    int64_t unk320;            // 320
-    int64_t unk328;            // 328
-    int64_t unk330;            // 330
-    int64_t unk338;            // 338
-    int32_t unk340;            // 340
-    int64_t unk348;            // 348
+    int64_t unk2F0;                       // 2F0
+    int64_t unk2F8;                       // 2F8
+    int32_t unk300;                       // 300
+    int32_t unk304;                       // 304
+    CGameFramework* framework;            // 308
+    services::GameServices* gameServices; // 310
```

Comment:
For my info, what do `GameServices` do?

Reply:
Given the actual class set on PC seems to be `services::GameServicesWin` and it's casted to `services::GameServices`, I assume it's an abstraction layer for some platform specific stuff
At the very least, it checks if the expansion is installed